src/js/App/NoAccess.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import {
+    Title,
+    Button,
+    EmptyState,
+    EmptyStateVariant,
+    EmptyStateIcon,
+    EmptyStateBody
+} from '@patternfly/react-core';
+
+import { LockIcon } from '@patternfly/react-icons';
+
+const NoAccess = ({ appName }) => {
+    appName = appName.charAt(0).toUpperCase() + appName.slice(1);
+    const headerClass = document.querySelector('.appTitle');
+    headerClass.innerHTML = appName;
+    return (
+        <div>
```

Comment:
is there a reason you're wrapping the page with an empty `div`?

Reply:
that `div` is really sneaky... thanks!